libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFSDKLoginViewControllerConfig.m

```diff
@@ -40,8 +47,13 @@ - (instancetype)init {
         _navBarTextColor = [UIColor whiteColor];
         _showNavbar = YES;
         _showSettingsIcon = YES;
+        _appConfig = [[SFSDKAppConfig alloc] init];
```

Comment:
Given that initializing `SalesforceSDKManager` is a required step, this should probably be fetched from `SalesforceSDKManager` -> it has an `appConfig` property.

Reply:
@bhariharan Good catch. Oversight from my end.

Reply:
This would have to come from the property in `SalesforceSDKManager`.